editor/plugins/script_editor_plugin.cpp

```diff
@@ -92,7 +92,7 @@ Ref<EditorSyntaxHighlighter> EditorSyntaxHighlighter::_create() const {
 }
 
 void EditorSyntaxHighlighter::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("_get_edited_resource"), &EditorSyntaxHighlighter::_get_edited_resource);
+	ClassDB::bind_method(D_METHOD("get_edited_resource"), &EditorSyntaxHighlighter::_get_edited_resource);
```

Comment:
What is this needed for? Also this is probably called somewhere yet you haven't changed the places it was called in, this was intended to be internal so it would be called from some editor method or it wouldn't have been bound in the first place